templates/coredns/0/rancher-compose.yml

```diff
@@ -0,0 +1,84 @@
+.catalog:
+  name: "CoreDNS"
+  version: "1.0.1"
+  minimum_rancher_version: v1.6.14
+  description: "CoreDNS is a DNS server that chains plugins and provides such DNS Services like Etcd & Kubernetes. For kubernetes, please use helm coredns's chart."
+  questions:
+    - variable: "ZONES"
+      label: "DNS zone names"
+      description: "The name of an existing zone in which to create the records."
+      type: "string"
+      default: "rancher.io"
+      required: true
+    - variable: "ROOT_PATH"
+      label: "Etcd root path"
+      description: "Etcd root path which is used to save records."
+      default: "/skydns"
+      type: "string"
+      required: true
+    - variable: "ETCD_ENDPOINTS"
+      label: "Etcd endpoints"
+      description: "Etcd service endpoints."
+      type: "string"
+      default: "http://localhost:2379"
+      required: true
```

Comment:
Inside container will be already running an etcd service?? If not, default value wouldn't work...
May be it'd be better to leave blank default value, to force users to set the correct etcd endpoints. btw, multiple values are allowed??